code/components/extra-natives-five/src/PoolTraversalNatives.cpp

```diff
@@ -291,6 +307,23 @@ static InitFunction initFunction([]()
 			throw std::runtime_error(va("Invalid pool: %s", pool));
 		}
 	});
+
+	fx::ScriptEngine::RegisterNativeHandler("GET_GAME_POOL_SIZE", [](fx::ScriptContext& context)
+	{
+		std::string pool = context.CheckArgument<const char*>(0);
+		if (pool.compare("CPed") == 0)
```

Comment:
We're using brackets even for single-line blocks.

```cpp
if (pool.compare("CPed") == 0)
{
  GetPoolSize<PedPoolTraits>(context);
} 
else if (pool.compare("CObject") == 0)
{
// etc...
```

Reply:
I knew this would come up - so, do you want us to **deviate** from already existing implementations? Previously this has been frowned upon.

https://github.com/citizenfx/fivem/blob/c73d37249419fa9ef06947652a6a2065d0057e24/code/components/extra-natives-five/src/PoolTraversalNatives.cpp#L294

Reply:
If so, why .compare() and not string_view, etc. It opens up for further improvements.. But do we _not_ fix the other (almost identical) native?

Reply:
Iirc there was some PR that meant to rework the related a bit, we also suggested to add brackets. Not sure what happened to that PR...

Reply:
I get it, but as active contributors having clear guidelines on how to proceed with this in the future would be much appreciated.

Do we **follow** already existing code and style, even if wrongly implemented?
Do we only follow the guide line for the new code we add and live with the old, non-compliant code?
Do we implement the new code as per the style guide and also **fix** any related natives that are also incorrect?

Reply:
> Do we follow already existing code and style, even if wrongly implemented?

Usually this, yes.